src/pages/workspace/companyCards/WorkspaceCompanyCardDetailsPage.tsx

```diff
@@ -0,0 +1,197 @@
+                            >
+                                <MenuItemWithTopDescription
+                                    description={translate('workspace.moreFeatures.companyCards.cardName')}
+                                    title={card?.nameValuePairs?.cardTitle}
+                                    shouldShowRightIcon
+                                    brickRoadIndicator={card?.nameValuePairs?.errorFields?.cardTitle ? CONST.BRICK_ROAD_INDICATOR_STATUS.ERROR : undefined}
+                                    onPress={() => Navigation.navigate(ROUTES.WORKSPACE_COMPANY_CARD_NAME.getRoute(policyID, cardID, bank))}
+                                />
+                            </OfflineWithFeedback>
+                            {exportMenuItem && (
+                                <OfflineWithFeedback
+                                    pendingAction={card?.nameValuePairs?.pendingFields?.exportAccountDetails}
+                                    errorRowStyles={[styles.ph5, styles.mb3]}
+                                    errors={ErrorUtils.getLatestErrorField(card?.nameValuePairs ?? {}, 'exportAccountDetails')}
+                                    onClose={() => Policy.clearCompanyCardErrorField(workspaceAccountID, cardID, bank, 'exportAccountDetails')}
+                                >
+                                    <MenuItemWithTopDescription
+                                        description={exportMenuItem.description}
+                                        title={exportMenuItem.title}
+                                        shouldShowRightIcon
+                                        onPress={() => Navigation.navigate(ROUTES.WORKSPACE_COMPANY_CARD_EXPORT.getRoute(policyID, cardID, bank))}
+                                    />
+                                </OfflineWithFeedback>
+                            )}
+                            <MenuItemWithTopDescription
+                                shouldShowRightComponent={card?.isLoadingLastUpdated}
+                                rightComponent={
+                                    <ActivityIndicator
+                                        style={[styles.popoverMenuIcon]}
+                                        color={theme.spinner}
+                                    />
+                                }
+                                description={translate('workspace.moreFeatures.companyCards.lastUpdated')}
+                                title={card?.isLoadingLastUpdated ? translate('workspace.moreFeatures.companyCards.updating') : card?.lastUpdated}
+                                interactive={false}
+                            />
+                            <MenuItemWithTopDescription
+                                description={translate('workspace.moreFeatures.companyCards.transactionStartDate')}
+                                title={card?.startDate ? format(card.startDate, CONST.DATE.FNS_FORMAT_STRING) : ''}
```

Comment:
It was causing [this issue](https://github.com/Expensify/App/issues/51915). More explained [here](https://github.com/Expensify/App/issues/51915#issuecomment-2462406800)